src/js/attendee.client.jsx

```diff
@@ -42,20 +43,30 @@ const redirectAfterLogin = () => {
     AuthService.removeListener('profile_updated', redirectCallback)
   }
 };
+BrandStore.on('receivedBranding', () => {
+  let branding = BrandStore.branding;
+  let myStyle = document.styleSheets[6];
+  myStyle.insertRule(".primaryText { color: " + branding.color_primary + " !important }", 0);
```

Comment:
This is super neat, is it the prefered style for programatically changing style?